biztalk/core/how-to-configure-transport-advanced-options-for-a-send-port.md

```diff
@@ -23,6 +23,13 @@ manager: "anneta"
 # How to Configure Transport Advanced Options for a Send Port
 Use the BizTalk Server Administration console to configure transport advanced options for a send port. These options determine how messages are handled by the send port, such as the number of times to retry sending messages on message failure and the service window schedule for the port.  
   
+> [!NOTE]
+> **Starting with BizTalk Server 2020**, dynamic send ports with ordered delivery can process messages to different outbound locations in parallel, allowing for higher throughput. Order will be maintained per unique outbound location, but not across different outbound locations even for the same transport type.
+>
+> Consider six messages: M1, M2, M3, M4, M5, and M6. M1, M3, M5 are meant for a file location F1. M2, M4, and M6 are meant for a file location F2. The ordered delivery dynamic send port makes sure that M1, M3, and M5 are ordered; and M2, M4, and M6 are ordered respectively.
+>
+> You can choose to enforce order across all outbound locations for a given transport type by changing the 'Enforce order across outbound locations' setting on the 'Transport Advanced Options' tab.
+
```

Comment:
Would it be better if this section is after the "ordered delivery section" below, because the new "Enforce order across outbound locations" are based on ordered delivery. and for the six messages example, it would be nice to distinguish how these 2 properties affect them differently.
I would say:
Consider the 6 messages scenario above, if these 6 messages are for 2 file locations F1 and F2, M1 will has to been sent first, then M2 sent by F2, and then M3 sent by F1, and M4 sent by F2 , with it selected, you will get M1,M3,M5 sent by F1 and M2, M4 and M6 sent by F2 in parallel. 

I might understand this incorrectly. but this explain what i mean 1) Better not to duplicate the example statements 2)distinguish with ordered delivery.

Reply:
Not sure I understand.
The 2016 note is for ordered delivery of messages for different adapters. The 2020 note is for ordered delivery within the same adapter but different addresses. I would not want to refer to the 'above' scenario in that case as it is a completely different scenario. We can discuss

Reply:
Updated